### ISSUE TITLE
General data cleaning scripts not related to Spectral

### ISSUE THREAD

**Issue by cnske** [fieldSITES/scripts#1]
Hej there!

I started to work on different data cleaning scripts, e.g. for our manual profiling and would love to add them to the repo, not only for making them accessible to all of you guys, but also to get possible feedback and open the option for edge case testing at other stations. 

Where should such scripts be stored?

**Comment by jobelund** [comment]
I suggest is to create the subfolder fieldSITES/<THEMATIC_PROGRAM>/scripts/<YOUR_REPO_NAME>
:)

**Comment by jonathanthiry** [comment]
Yes, any new folder next to the UAV, Phenocam, etc folders would work. Please create a pull request if you would like feedback.

**Comment by cnske** [comment]
Thanks. Sounds like a good option. I will also add the station name and adjust the [README.md](https://github.com/fieldSITES/scripts/blob/main/README.md) to add some documentation.
